src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs

```diff
@@ -193,7 +193,10 @@
             (when (pos? count-all)
               [:ul {:style {:marginTop "0"}}
                (for [[status subs] (sort submissions-count)]
-                 [:li {} (str subs " " status)])])]))]]
+                 [:li {} (str subs " " status)])])]))
+       ]]
+    (when editing? [comps/PHIWarning])
```

Comment:
Formatting.  The `]]` should go back on the previous line, and this isn't indented right.